### PR TITLE
Better early return for add_launch_buttons if no launch buttons

### PR DESCRIPTION
**Pull request by paugier**
Fixes https://github.com/executablebooks/jupyter-book/issues/1463

See https://github.com/executablebooks/jupyter-book/issues/1463#issuecomment-1445131176

Is it enough?

### CODE REVIEW

**Comment by choldgraf** [comment]
Nice - this looks correct to me. Could you just add a test to make sure that if none if them are there, then there is no launch buttons dropdo container on the page?

File: src/sphinx_book_theme/header_buttons/launch.py [hunk 1]
```diff
@@ -41,7 +41,14 @@ def add_launch_buttons(
     # If so, insert the URLs depending on the configuration
     config_theme = app.config["html_theme_options"]
     launch_buttons = config_theme.get("launch_buttons", {})
-    if not launch_buttons or not _is_notebook(app, pagename):
+    if (
+        not launch_buttons
+        or not _is_notebook(app, pagename)
+        or not any(
+            launch_buttons[key]
```
Comment (choldgraf) [inline src/sphinx_book_theme/header_buttons/launch.py:48]: Shouldnt we use `launch_buttons.get(key)` in case one is not present so we don't raise a key error?

**Comment by choldgraf** [comment]
actually, no worries I will add a test in #691 - many thanks for this improvement :-)
